pquery3-run.conf

```diff
@@ -0,0 +1,256 @@
+################################################################################
+RUNDIR=/tmp/$RANDOMD
+
+# Sysbench data load. Normally set to 0 (off)
+SYSBENCH_DATALOAD=0
+
+################################################################################
+#              User configurable variables: generics 2                         #
+################################################################################
+
+################################################################################
+# Number of individual trials to execute (one can always interrupt with ctrl+c #
+################################################################################
+TRIALS=3
+
+################################################################################
+# Add random mysqld --options to MYEXTRA using an input file. 1=On, 0=off.     #
+# (Auto-disabled if PXC=1)                                                     #
+################################################################################
+ADD_RANDOM_OPTIONS=1
+
+################################################################################
+# Add random tokudb --options to MYEXTRA using an input file. 1=On, 0=off.     #
+# (Auto-disabled if PXC=1)                                                     #
+################################################################################
+ADD_RANDOM_TOKUDB_OPTIONS=0
+
+################################################################################
+# Max nr of random options to add (minimum is always 1).                       #
+# Recommended: 4. Counts per ADD_RANDOM_... option above (x2)                  #
+################################################################################
+MAX_NR_OF_RND_OPTS_TO_ADD=5
+
+################################################################################
+# Save only trials that generate a core file (good for initial few runs where  #
+# there are lot of crashes/asserts)                                            #
+################################################################################
+SAVE_TRIALS_WITH_CORE_OR_VALGRIND_ONLY=0
+
+################################################################################
+# Saves per-trial SQL even if SAVE_TRIALS_WITH_CORE_OR_VALGRIND_ONLY=1.        #
+# Main usecase: full server lockups/hangs                                      #
+################################################################################
+SAVE_SQL=0
+
+################################################################################
+# Store a copy of the INFILE SQL input file in the work directory (not related #
+# to per-trial SQL)                                                            #
+################################################################################
+STORE_COPY_OF_INFILE=0
+
+################################################################################
+# x sec max trial runtime within which pquery tries to process                 #
```

Comment:
give some comment i.e. with every step runtime will increase.

Reply:
Added comments in the conf file. Also added console logs about it